### PR TITLE
updated readme.md file

### PR DESCRIPTION
**Pull request by kumarharsh2396**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,5 +11,82 @@ cd build
...
+
+![banner](https://github.com/kumarharsh2396/passgen-cpp/blob/main/_%F0%9F%94%90_%E2%89%AA_passgen-cpp_%E2%89%AB_%F0%9F%85%92%F0%9F%85%9B%F0%9F%85%98_tool.png?raw=true)
+
+ A CLI password generator, written in C++.
+
+ It generates random passwords according to the configurations provided by the user, such as the length, which set of characters to include in the password, etc. 
+
+
+# Installing Dependencies :-
+ It uses cmake to build the programm so it needs to be installed on system.
```
Comment (anjanik012) [inline README.md]:
```suggestion
 It uses cmake to build the program so it needs to be installed on system.
```

File: README.md [hunk 2]
```diff
@@ -11,5 +11,82 @@ cd build
...
+
+to know more about cmake refer to this page :- [cmake](https://cmake.org/cmake/help/latest/guide/tutorial/index.html)
```
Comment (anjanik012) [inline README.md]:
```suggestion
To know more about cmake refer to this page :- [https://cmake.org/cmake/help/latest/guide/tutorial/index.html](https://cmake.org/cmake/help/latest/guide/tutorial/index.html)
```

File: README.md [hunk 3]
```diff
@@ -11,5 +11,82 @@ cd build
...
+
+ cmake can be installed using the following commands
```
Comment (anjanik012) [inline README.md]:
```suggestion
 Cmake can be installed using the following commands
```

File: README.md [hunk 4]
```diff
@@ -11,5 +11,82 @@ cd build
...
+
+
+
+```cmake
+sudo apt-get install cmake
```
Comment (anjanik012) [inline README.md:44]: Please add instructions for more Linux distributions

File: README.md [hunk 5]
```diff
@@ -11,5 +11,82 @@ cd build
...
+```
+# Build instructions
+
+
+- cloning this repository
```
Comment (anjanik012) [inline README.md]:
```suggestion
- Cloning this repository
```

File: README.md [hunk 6]
```diff
@@ -11,5 +11,82 @@ cd build
...
+```
+$ git clone https://github.com/hnccbits/passgen-cpp.git
+```
+- going into the reposistory
```
Comment (anjanik012) [inline README.md]:
```suggestion
- Going into the reposistory
```

File: README.md [hunk 7]
```diff
@@ -11,5 +11,82 @@ cd build
...
+```
+$ cd passgen-cpp
+```
+- generating standard build files
```
Comment (anjanik012) [inline README.md]:
```suggestion
- Generating standard build files
```

File: README.md [hunk 8]
```diff
@@ -11,5 +11,82 @@ cd build
...
+$ cmake -B build/
+```
+```
+$ cd build
+$ make
+```
+
+# Installation
+
+In the passgen-cpp
```
Comment (anjanik012) [inline README.md]:
```suggestion
In the project root while generating CMake build files
```

File: README.md [hunk 9]
```diff
@@ -11,5 +11,82 @@ cd build
...
+
+```
+$ cmake -B build/ -DCMAKE_INSTALL_PREFIX="/usr"
+$ sudo make install
+```
+
+# Usage
+ Run the following command in  the ``` build ``` directorry for information on usage
```
Comment (anjanik012) [inline README.md]:
```suggestion
 Run the following command for information on usage
```

File: README.md [hunk 10]
```diff
@@ -11,5 +11,82 @@ cd build
...
+ ```
+ $ passgen -H
+ ```
+ ![usage](https://github.com/kumarharsh2396/passgen-cpp/blob/main/carbon.png?raw=true)
+
+
+# Features
```
Comment (anjanik012) [inline README.md]: Move this above, it shouldn't be in the bottom

File: README.md [hunk 11]
```diff
@@ -11,5 +11,82 @@ cd build
...
+
+```
+$ cmake -B build/ -DCMAKE_INSTALL_PREFIX="/usr"
+$ sudo make install
+```
+
+# Usage
+ Run the following command in  the ``` build ``` directorry for information on usage
+ ```
+ $ passgen -H
```
Comment (anjanik012) [inline README.md:83]: Add usage instructions here as well along with this command

File: README.md [hunk 12]
```diff
@@ -11,5 +11,82 @@ cd build
 make
 ```
 
-[Read Design goals](DESIGN.md)
+[Read Design goals](DESIGN.md) -->
+![license](https://img.shields.io/badge/license-MIT-brightgreen) 
+![license](https://img.shields.io/badge/dependencies-cmake-yellowgreen)
+![license](https://img.shields.io/badge/forks-13-blue)
+
+![banner](https://github.com/kumarharsh2396/passgen-cpp/blob/main/_%F0%9F%94%90_%E2%89%AA_passgen-cpp_%E2%89%AB_%F0%9F%85%92%F0%9F%85%9B%F0%9F%85%98_tool.png?raw=true)
```
Comment (anjanik012) [inline README.md]: All links should point to our repository and not a fork, correct this

File: README.md [hunk 13]
```diff
@@ -11,5 +11,82 @@ cd build
...
+
+ A CLI password generator, written in C++.
+
+ It generates random passwords according to the configurations provided by the user, such as the length, which set of characters to include in the password, etc. 
+
+
+# Installing Dependencies :-
```
Comment (anjanik012) [inline README.md]:
```suggestion
# Installing Build Dependencies :-
```

**Review by anjanik012 — CHANGES_REQUESTED** [review]

File: README.md (hunk 12 above)
Comment (kumarharsh2396) [reply README.md]: But this img file is not present in our repository.

Comment (anjanik012) [reply README.md]: add it

Comment (kumarharsh2396) [reply README.md]: Okk sir

File: README.md (hunk 10 above)
Comment (kumarharsh2396) [reply README.md]: where do i move it usage should not be above installation

Comment (anjanik012) [reply README.md]: Every software first tells about the features and later other things

Comment (kumarharsh2396) [reply README.md]: okk sir

**Pull request merged**